plugins/board/src/index.ts

```diff
@@ -111,6 +112,12 @@ export interface MenuPage extends Doc {
   label: IntlString
 }
 
+/**
+ * @public
+ */
+export interface LabelsCompactMode extends Preference {
```

Comment:
I am not quite sure how this preference mechanism is working, but shouldn't we have just a preference for Board application per each user and store all possible preferences there?

Reply:
Preference object are individual per user, so any kind of model is possible.